plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -293,6 +239,51 @@ protected void checkInformationSchemaViewsForMaterializedView(String schemaName,
                 .hasMessageFindingMatch("(?s)Expecting.*to contain:.*\\Q[(" + viewName + ")]");
     }
 
+    // Override is required because metadata deletes do not return the number of rows deleted. See todo in IcebergMetadata#executeDelete
+    @Test
+    @Override
+    public void testDeleteWithComplexPredicate()
+    {
+        skipTestUnlessSupportsDeletes();
```

Comment:
redundant (copy leftover) -- this class knows that deletes are supported

same below

Reply:
The assertions are different from the one in the base class. Related to https://github.com/trinodb/trino/pull/11886#discussion_r853884254